letsencrypt/cli.py

```diff
@@ -704,12 +706,18 @@ def obtain_cert(config, plugins, lineage=None):
 
     if config.dry_run:
         _report_successful_dry_run()
-    elif config.verb == "renew" and installer is not None:
-        # In case of a renewal, reload server to pick up new certificate.
-        # In principle we could have a configuration option to inhibit this
-        # from happening.
-        installer.restart()
-        print("reloaded")
+    elif config.verb == "renew":
+        if installer is None:
+            # Tell the user that the server was not restarted.
+            print("new certificate deployed without reload, fullchain is",
+                  lineage.fullchain)
+        else:
+            # In case of a renewal, reload server to pick up new certificate.
+            # In principle we could have a configuration option to inhibit this
+            # from happening.
+            installer.restart()
+            print("new certificate deployed with reload of plugin",
```

Comment:
Perhaps `"new certificate deployed with reload of ", config.installer.name, " server; fullchain is ", lineage.fullchain`